numba/cuda/api.py

```diff
@@ -85,6 +85,14 @@ def is_cuda_array(obj):
     return hasattr(obj, '__cuda_array_interface__')
 
 
+def is_float16_supported():
+    """Whether 16-bit floats are  supported.
+
+        Returns True if float16 is supported, False otherwise.
```

Comment:
The indentation is out-of-place here:

```suggestion
    Returns True if float16 is supported, False otherwise.
```